sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -578,3 +581,19 @@ def set_read_only(
             raise HttpResponseError(message=error.message, response=error.response)
         except binascii.Error:
             raise binascii.Error("Connection string secret has incorrect padding")
+
+    def update_sync_token(self, token):
+        # type: (str) -> None
+
+        """Add a sync token to the internal list of tokens.
+        :param token: The sync token to be added to the internal list of tokens
+        :type token: str
+        """
+
+        self._sync_token_policy.add_token(token)
```

Comment:
Given you have self._sync_token_policy = None above, do we need to check if self._sync_token_policy before calling add?

Reply:
Yes, good catch. If the user provides a `pipeline` kwarg when instantiating the client, they are giving up the ability to use sync tokens.